server/auth/index.js

```diff
@@ -0,0 +1,34 @@
+const router = require("express").Router();
+const {
+  models: { User },
+} = require("../db");
+module.exports = router;
+
+router.post("/login", async (req, res, next) => {
+  try {
+    res.send({ token: await User.authenticate(req.body) });
+  } catch (err) {
+    next(err);
+  }
+});
+
+router.post("/signup", async (req, res, next) => {
+  try {
+    const user = await User.create(req.body);
```

Comment:
YESSS I CAN HACK YOUR WEBSITE!
this is very bad practice `const user = await User.create(req.body);`
the reason is because... you have an `isAdmin` attribute in your User table
that means i can just postman (or curl) request to manually set the `isAdmin` column to "true" for my user
how to fix?

you can just do ```const user = await User.create({ firstName: req.body.firstName, lastName: req.body.lastName, email: ... });```